FirebaseAuth/Sources/Public/FirebaseAuth/FIRAuth.h

```diff
@@ -858,6 +858,16 @@ NS_SWIFT_NAME(Auth)
 - (void)revokeTokenWithAuthorizationCode:(NSString *)authorizationCode
                               completion:(nullable void (^)(NSError *_Nullable error))completion;
 
```

Comment:
Maybe add a comment on why this macro check is needed?

Reply:
Yeah, will make a custom micro with detailed comment in a separate pr as discussed with Paul above. It applies to other existing micros as well.